packages/design-system-docs/src/pages/components/Table/_Table.docs.scss

```diff
@@ -17,9 +19,84 @@ Table with two headers and a caption
 
 Markup: table-complex.example.html
 
+.ds-c-table--striped - Striped table
```

Comment:
I see you added the stripe class to the existing complex example. I think we should update our Table examples to be consistent with other pages, i.e. [the button page](http://localhost:3000/components/button/)

I've never told you any of this context or these design decisions, but basically this page uses our legacy design for examples, where we use the `modifier` tag to automatically generate documentation for variants. You can see that [here](https://github.com/CMSgov/design-system/pull/706/files/818b6597b1862b4d9e7ae812b44475216a1ea71f#diff-c408290eec1f1b2fbb29243804f68058L10) and in `table.example.html`, which uses EJS templating to generate documentation. 

In the past, we revamped most of our pages to use the new pattern, where we try to have only 1 larger example file, and label them instead of just showing the class. 

You can see an example of this by comparing this [legacy style](https://design.cms.gov/style/base/) with the [newer style](https://design.cms.gov/utilities/background-color/)

Reply:
Basically, I think it makes sense to update `table.example.html` to contain the default, borderless, and striped variants inside a single, large table. Then we can keep the complex table example the same as a separate section, as its technically not a variant, but just a different more complex use case. Another example of this is the [button with icons](https://design.cms.gov/components/button/#components.button.icons), they have their own separate example file because they dont actually have a separate variant classname, and they are just a different use case of the default styling.

Reply:
Good suggestion.

Reply:
Updated `table.example.html` to contain the through variants.